content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
+* **The output of those above examples are compiled files.** The CSS files are generated by processing the source SCSS. The HTML files are generated by processing the pseudo-HTML source. And all pages are generated from exampleSite contents. All are generated from source files and will be overwritten if the source is compiled again. Output files can be found on `exampleSite/public` folder, when you run `hugo --themesDir ../..` this command. If you want the minified output, then run `hugo --minify --themesDir ../..`.
+
+---
+
+## Working with source files
+
+Now that you’ve got your theme running and understand the compiling process, let’s discuss the proper ways to edit and customize your theme. You can directly customize your template from the source file. For a better experience, open the whole theme folder with a text-editor or IDM (i.e. [Visual Studio Code](https://code.visualstudio.com/)).
 
- * `preloader` is for site preloader. It starts with `[params.preloader]`. you can disable preloader by `enable = false`. If you want to show any image, logo, or animation in preloader then give the directory in `preloader` (i.e. `preloader = images/preloader.gif`), or if you don't want any logo or image then leave the preloader empty, then it will show `theme body color` screen when every component is loading. that's simple.
+### Site configuration
 
- * `social` is for the social sites that show on your theme. It starts with `[params.social]`. It's a loop, so you can add more social button by copying its previous loop item, and change the value. Or you can remove a social button by removing this loop item. Most of the themes use [themify icon pack](https://themify.me/themify-icons). And the rest of the themes uses [Fontawesome icon pack](https://fontawesome.com/v4.7.0/icons/). If icon starts with `ti` then you can understand that it uses the Themify icon pack, or if it starts with `fa` then it uses font awesome icon pack.
+You have a `config.toml` file in the `exampleSite` folder. Which gives you the ability to configure your site, such as `changing logo` or `customize menu`.
 
- * `Language` is for multilingual content. we basically added two languages in a multilingual theme. you can add more language following this example content.
+#### Default configuration
+
+Default configuration is those which come with the Hugo library. Remember that, all configuration does not come with every theme.
+
+* `baseURL` defines your site URL. Give your site URL here (i.e. https://examplesite.org).
+* `title` is for your site title. Which shown on the browser tab.
+* `theme` name is already provided with every theme. don’t need to customize this theme name.
+* `summaryLeangth` is for the post excerpt limit. you can set your post excerpt length as you want to show.
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)